website/www/site/content/en/contribute/_index.md

```diff
@@ -57,46 +57,100 @@ There are lots of opportunities to contribute. You can for example:
  https://cwiki.apache.org/confluence/display/BEAM/Contributor+FAQ)
```

Comment:
(what is a Beam pipeline and how does it run?) -> (read more about what a Beam pipeline is and how it runs [here in Documentation](https://beam.apache.org/documentation/basics/))

Reply:
changed